centos.sh

```diff
@@ -102,49 +104,49 @@ generate_new_ramdisk() {
+      echo -e "" >> "$MODULESFILE" 2>> "$DEBUGFILE"
 
-      echo -e "# hdds" >> $MODULESFILE 2>>$DEBUGFILE
+      echo -e "# hdds" >> "$MODULESFILE" 2>> "$DEBUGFILE"
       HDDDEV=""
       for hddmodule in $MODULES; do
-        if [ "$hddmodule" != "powernow-k8" -a "$hddmodule" != "via82cxxx" -a "$hddmodule" != "atiixp" ]; then
-          echo -e "alias scsi_hostadapter$HDDDEV $hddmodule" >> $MODULESFILE 2>>$DEBUGFILE
+        if [ "$hddmodule" != "powernow-k8" ] && [ "$hddmodule" != "via82cxxx" ] && [ "$hddmodule" != "atiixp" ]; then
+          echo -e "alias scsi_hostadapter$HDDDEV $hddmodule" >> "$MODULESFILE" 2>> "$DEBUGFILE"
           [ -z "$HDDDEV" ] && HDDDEV="1" || HDDDEV="$(echo $[$HDDDEV+1])"
         fi
       done
-      echo -e "" >> $MODULESFILE 2>>$DEBUGFILE
-    elif [ $IMG_VERSION -ge 60 ] ; then
+      echo -e "" >> "$MODULESFILE" 2>> "$DEBUGFILE"
+    elif [ "$IMG_VERSION" -ge 60 ] ; then
       # blacklist some kernel modules due to bugs and/or stability issues or annoyance
       local blacklist_conf="$FOLD/hdd/etc/modprobe.d/blacklist-hetzner.conf"
-      echo -e "### Hetzner Online GmbH - installimage" > $blacklist_conf
-      echo -e "### silence any onboard speaker" >> $blacklist_conf
-      echo -e "blacklist pcspkr" >> $blacklist_conf
-      echo -e "### i915 driver blacklisted due to various bugs" >> $blacklist_conf
-      echo -e "### especially in combination with nomodeset" >> $blacklist_conf
-      echo -e "blacklist i915" >> $blacklist_conf
+      echo -e "### $COMPANY - installimage" > "$blacklist_conf"
+      echo -e "### silence any onboard speaker" >> "$blacklist_conf"
+      echo -e "blacklist pcspkr" >> "$blacklist_conf"
+      echo -e "### i915 driver blacklisted due to various bugs" >> "$blacklist_conf"
+      echo -e "### especially in combination with nomodeset" >> "$blacklist_conf"
+      echo -e "blacklist i915" >> "$blacklist_conf"
     fi
 
-    if [ $IMG_VERSION -ge 70 ] ; then
+    if [ "$IMG_VERSION" -ge 70 ] ; then
       DRACUTFILE="$FOLD/hdd/etc/dracut.conf.d/hetzner.conf"
-      echo "add_dracutmodules+=\"mdraid lvm\"" >> $DRACUTFILE
-      echo "add_drivers+=\"raid1 raid10 raid0 raid456\"" >> $DRACUTFILE
-      echo "mdadmconf=\"yes\"" >> $DRACUTFILE
-      echo "lvmconf=\"yes\"" >> $DRACUTFILE
-      echo "hostonly=\"no\"" >> $DRACUTFILE
-      echo "early_microcode=\"no\"" >> $DRACUTFILE
+      echo "add_dracutmodules+=\"mdraid lvm\"" >> "$DRACUTFILE"
+      echo "add_drivers+=\"raid1 raid10 raid0 raid456\"" >> "$DRACUTFILE"
+      echo "mdadmconf=\"yes\"" >> "$DRACUTFILE"
+      echo "lvmconf=\"yes\"" >> "$DRACUTFILE"
+      echo "hostonly=\"no\"" >> "$DRACUTFILE"
+      echo "early_microcode=\"no\"" >> "$DRACUTFILE"
     fi
 
-    if [ $IMG_VERSION -ge 70 ] ; then
+    if [ "$IMG_VERSION" -ge 70 ] ; then
       execute_chroot_command "/sbin/dracut -f --kver $VERSION"; EXITCODE=$?
     else
-      if [ $IMG_VERSION -ge 60 ] ; then
+      if [ "$IMG_VERSION" -ge 60 ] ; then
```

Comment:
I don't think so, than the commands would also run on every CentOS7, but there we only want the stuff from line 147